docs/labor/3-felhasznaloi-felulet/index.md

```diff
@@ -765,6 +765,7 @@ Nem jelenik meg a listában az új elem, mert a `ListView` nem értesül arról,
 ```csharp
 public ObservableCollection<Person> People { get; set; }
 ```
+Ne felejtsük el megváltoztatni a MainWindow()-ban a listát inicializáló sort!
```

Comment:
ki tudnád inkább egyészíteni a fenti kódblokkot légyszi, hogy a konstruktort is tartalmazza? `hl_lines=""` paraméterrel lehet kiemelni sorokat is